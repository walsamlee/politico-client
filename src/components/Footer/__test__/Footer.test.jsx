import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../Footer';

describe('<Footer />', () => {
  it('should render Footer component', () => {
    const wrapper = shallow(
      <Footer />,
    );
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('div'));
  });
});
