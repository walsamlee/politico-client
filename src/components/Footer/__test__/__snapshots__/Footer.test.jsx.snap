// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render Footer component 1`] = `ShallowWrapper {}`;
